test/integration/read-property-multiple.spec.js

```diff
@@ -25,16 +25,16 @@ describe('bacstack - readPropertyMultiple integration', function() {
     client.readPropertyMultiple('127.0.0.1', requestArray, function(err, response) {
       var object = utils.propertyFormater(response.values[0].values);
       expect(response.values[0].objectIdentifier).to.deep.equal({type: 29, instance: 31});
-      expect(object[28]).to.deep.equal([{value: 'Collection of lighting presence detector', type: 7}]);
+      expect(object[28]).to.deep.equal([{value: 'Collection of lighting presence detector', type: 7, encoding: 0}]);
```

Comment:
Well, but if we change the value from `string` to `{}` it will also be a breaking change.